templates/security/cve/_cve-table.html

```diff
@@ -2,20 +2,30 @@
   <thead>
     <tr>
       <th style="width: 10em;">ID</th>
-      <th>Priority</th>
-      <th>Package</th>
-      {% for release in releases %}
-        <th>
-          <div class="icon-container__text">
-            {{ release.version }} {{ release.support_tag }}
-          </div>
-        </th>
-      {% endfor %}
+      <th style="width: 8em;">Priority</th>
+      <th style="width: 8em;">Package</th>
+      {% if version_releases %}
+        {% for version in version_releases %}
+          <th style="width: 8em;">
+            <div class="icon-container__text">
+              {{ version.version }} {{ version.support_tag }}
+            </div>
+          </th>
+        {% endfor %}
+      {% else%}
+        {% for release in releases %}
+          <th style="width: 8em;">
+            <div class="icon-container__text">
+              {{ release.version }} {{ release.support_tag }}
+            </div>
+          </th>
+        {% endfor %}
+      {% endif %}
```

Comment:
Here, these two loops are literally identical right? except built off different lists. Could we rename `version_releases` to `selected_releases` and always build this block off that?

Reply:
(and then rename `releases` to `all_releases`, for building the form, to make it clear that it's never the filtered set)